sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/ResourceBase.java

```diff
@@ -158,6 +159,17 @@ protected Response fetchRows(
     return resultRows;
   }
 
+  protected static ArrayNode convertRowsToArrayNode(QueryOuterClass.ResultSet rs) {
```

Comment:
Demoted from "Keyspaces" endpoint as it is now used from multiple places. Could conceivably move to shared codec class, but since our future wrt gRPC/Bridge is uncertain did not refactor further.